xla/service/cpu/runtime_fft_impl.h

```diff
@@ -262,6 +262,9 @@ void EigenFftImpl(const EigenDevice& device, void* out, void* operand,
       abort();
   }
 }
+void DuccFftImpl(void *out, void *in, internal::FftType fft_type,
```

Comment:
For single threaded, it will just be a dummy default, right?

We'll do threadpool work in subsequent PR.

Reply:
In order to set the actual threadpool via `set_active_pool`, you will need access to XLA's existing runtime intra-op pool, which you need to get from the runtime device.  That should probably be done within the implementation, rather than outside.